<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MntpOptDialog</class>
 <widget class="QDialog" name="MntpOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mountpoint Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="Label15">
     <property name="text">
      <string>Carrier</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLineEdit" name="lESourceTable16"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lESourceTable7"/>
   </item>
   <item row="14" column="1">
    <widget class="QComboBox" name="lESourceTable14">
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="Label4">
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="lESourceTable10">
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Label3">
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lESourceTable2"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lESourceTable3"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Label16">
     <property name="text">
      <string>Generator</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>&amp;Ok</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="Label13">
     <property name="text">
      <string>Bitrate</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lESourceTable5"/>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lESourceTable9"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lESourceTable8"/>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="lESourceTable12"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="Label8">
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="Label10">
     <property name="text">
      <string>Compr-encryp</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="Label7">
     <property name="text">
      <string>Nav-System</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="Label2">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lEMountPoint"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="Label6">
     <property name="text">
      <string>Solution</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="lESourceTable4">
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QComboBox" name="lESourceTable11">
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Label14">
     <property name="text">
      <string>Source ID</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="Label9">
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lESourceTable6"/>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="lESourceTable13"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Label1">
     <property name="text">
      <string>Format Details</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="Label12">
     <property name="text">
      <string>Free</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lESourceTable1"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="Label11">
     <property name="text">
      <string>Auth</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QComboBox" name="lESourceTable15">
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Y</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Label17">
     <property name="text">
      <string>Mountpoint</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
